### PR TITLE
python3Packages.advocate: init at 1.0.0

### PR DESCRIPTION
**Pull request by pborzenkov**
###### Description of changes

Init `python3Packages.advocate` at 1.0.0. This is a requirement to update `calibre-web` to 0.6.18

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/advocate/default.nix [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+buildPythonPackage rec {
+  version = "1.0.0";
+  pname = "advocate";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-G/EXDkEzQnmZZYAynFlOAXVAqw6vehUjI+dD8KhaNT0=";
+  };
+
```
Comment (gador) [inline pkgs/development/python-modules/advocate/default.nix:21]: Can we add an pythonImportsCheck ad least?

**Review by gador — CHANGES_REQUESTED** [review]
This is a semi-automatic executed nixpkgs-review with nixpkgs-review-checks extension. It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of nixpkgs-review pr 170802 run on x86_64-linux 1

<details>
<summary>
2 packages built:
</summary>
<ul>
<li>python310Packages.advocate</li>
<li>python39Packages.advocate</li>
</ul>
</details>

The following issues got detected with the above build packages.
Please fix at least the ones listed with your changed packages:

<details>
python310Packages.advocate:
<details>

got build log for '/nix/store/m1z50cjigqgj1d5bhpdysy34231c8fcc-python3.10-advocate-1.0.0' from 'daemon'
When evaluating attribute ‘python310Packages.advocate’:
warning: python-include-tests
Consider adding a checkPhase for tests, or if not feasible, pythonImportsCheck.

Near pkgs/development/python-modules/advocate/default.nix:38:

────────────────────
   |
38 |     description = "An SSRF-preventing wrapper around Python's requests library";
   | ^
────────────────────

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md

</details>
</details>

File: pkgs/development/python-modules/advocate/default.nix (hunk 1 above)
Comment (pborzenkov) [reply pkgs/development/python-modules/advocate/default.nix:23]: Done

File: pkgs/development/python-modules/advocate/default.nix [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+}:
+
+buildPythonPackage rec {
+  version = "1.0.0";
+  pname = "advocate";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-G/EXDkEzQnmZZYAynFlOAXVAqw6vehUjI+dD8KhaNT0=";
+  };
```
[the remark this answers is not in the document]
Comment (pborzenkov) [reply pkgs/development/python-modules/advocate/default.nix]: Done

**Review by gador — APPROVED** [review]

**Pull request merged**